src/Compilers/VisualBasic/Portable/Symbols/MethodSymbol.vb

```diff
@@ -1010,6 +1010,18 @@ Namespace Microsoft.CodeAnalysis.VisualBasic.Symbols
             End Get
         End Property
 
+        Private ReadOnly Property IMethodSymbol_CallingConvention As Reflection.Metadata.SignatureCallingConvention Implements IMethodSymbol.CallingConvention
+            Get
+                Return Cci.CallingConventionUtils.ToSignatureConvention(CallingConvention)
+            End Get
+        End Property
+
+        Public ReadOnly Property CallingConventionTypes As ImmutableArray(Of INamedTypeSymbol) Implements IMethodSymbol.CallingConventionTypes
+            Get
+                Return ImmutableArray(Of INamedTypeSymbol).Empty
```

Comment:
Not implementing this one but implementing the prior one -- is that just because that other one is easy enough to implement so it's a 'why not'?

Reply:
> Not implementing this one but implementing the prior one -- is that just because that other one is easy enough to implement so it's a 'why not'?

CallingConvention is applicable to regular methods 

---
In reply to: [463851342](https://github.com/dotnet/roslyn/pull/46467#discussion_r463851342) [](ancestors = 463851342)